### ISSUE TITLE
2 small issues with 1.0.6 version

### ISSUE THREAD

**Issue by handioq** [Tune389/Tap-Titans-2-Progress-Bot#5]
Hello. I've already posted this on the forum, but may be there u see this faster.

Hero levelup clicks works incorrectly as for me in v.1.0.6 (or may be it is expected behavior, I dont know):
Bot upgrade heroes from bottom to top, but first upgraged hero is second, but not first from this order. For now order is like 2nd from bottom, 1st from bottom and then 3rd from bottom. I think better order would be like this: 1st from bottom, 2nd from bottom, 3rd from bottom.

And there is (in v1.0.6) some additional opening of heroes leveling tab just before normall leveling. Its about 3-5 secs but I think may be that is some small bug.

First issue with hero up order i've fixed via this [code](https://github.com/Tune389/Tap-Titans-2-Progress-Bot/commit/c7665770e86308e4b4231983f4d92c6b60697dcb#diff-e35d7ac2842e0331a9508a914a7aa968R680):

```
        touchDown 1 466 720
	sleep 40
	touchUp 1
	touchDown 1 466 700
	sleep 40
	touchUp 1
	touchDown 1 466 680
```

May be it would be useful for somebody.

**Comment by handioq** [comment]
And additional information corresponding to second bug: this bug is reproducible only if enable clan quest setting is activated.

**Comment by Tune389** [comment]
Order fixed in 1.0.7
